src/diffusers/loaders/lora_base.py

```diff
@@ -206,6 +209,7 @@ def _fetch_state_dict(
     subfolder,
     user_agent,
     allow_pickle,
+    metadata=None,
```

Comment:
Why is it passed here?

Reply:
Because we pass `metadata` while loading the adapter:

https://github.com/huggingface/diffusers/blob/47cad58caade48ec9b6b7b66ec4e3c16dc4bdc56/src/diffusers/loaders/lora_pipeline.py#L213

which is, in turn, passed to this method if you step through it.